src/interfaces/appstore.js

```diff
@@ -139,14 +139,14 @@ module.exports = function(app) {
               })
           })
           .catch(error => {
+            console.log(error.message)
+            debug(error.stack)
+            res.status(500)
```

Comment:
For the known "you are not connected to the internet" class of errors the status code should probably be 200, this is not 500 server error. 

emptyAppStoreInfo has `storeAvailable: true`, which ought to be false I think.